app/http.php

```diff
@@ -0,0 +1,781 @@
+            }
+
+            if ($buildStdout === '') {
+                $buildStdout = 'Build Successful!';
+            }
+
+            $buildEnd = \time();
+            $build = [
+                'outputPath' => $outputPath,
+                'status' => 'ready',
+                'stdout' => \utf8_encode(\mb_substr($buildStdout, -4096)),
+                'stderr' => \utf8_encode(\mb_substr($buildStderr, -4096)),
+                'startTime' => $buildStart,
+                'endTime' => $buildEnd,
+                'duration' => $buildEnd - $buildStart,
+            ];
+
+            Console::success('Build Stage completed in ' . ($buildEnd - $buildStart) . ' seconds');
+        
+        } catch (Throwable $th) {
+            Console::error('Build failed: ' . $th->getMessage());
+            throw new Exception($th->getMessage(), 500);
+        } finally {
+            if (!empty($buildId)) {
+                $orchestration->remove($buildId, true);
+            }
+            $orchestrationPool->put($orchestration);
+        }
+
+        /** Create runtime server */
+        try {
+            $orchestration = $orchestrationPool->get();
+            /**
+             * Copy code files from source to a temporary location on the executor
+             */
+            $buffer = $device->read($outputPath);
+            if(!$device->write($tmpBuild, $buffer)) {
+                throw new Exception('Failed to copy built code to temporary location.', 500);
+            };
+    
+            /** 
+             * Launch Runtime 
+            */
+            $container = 'runtime-' . $runtimeId;
+            $secret = \bin2hex(\random_bytes(16));
+            $vars = \array_merge($vars, [
+                'INTERNAL_RUNTIME_KEY' => $secret
+            ]);
+    
+            $executionStart = \microtime(true);
+            $executionTime = \time();
+
+            $vars = array_map(fn ($v) => strval($v), $vars);
+
+            $orchestration
+                ->setCpus(App::getEnv('OPENRUNTIMES_FUNCTIONS_CPUS', '1'))
+                ->setMemory(App::getEnv('OPENRUNTIMES_FUNCTIONS_MEMORY', '256'))
+                ->setSwap(App::getEnv('OPENRUNTIMES_FUNCTIONS_MEMORY_SWAP', '256'));
+
+            $id = $orchestration->run(
+                image: $baseImage,
+                name: $container,
+                vars: $vars,
+                labels: [
+                    'openruntimes-id' => $runtimeId,
+                    'openruntimes-type' => 'function',
+                    'openruntimes-created' => strval($executionTime),
+                    'openruntimes-runtime' => $runtime
+                ],
+                hostname: $container,
+                mountFolder: \dirname($tmpBuild),
+            );
+
+            if (empty($id)) {
+                throw new Exception('Failed to create runtime', 500);
+            }
+
+            $orchestration->networkConnect($container, App::getEnv('OPENRUNTIMES_EXECUTOR_RUNTIME_NETWORK', 'openruntimes'));
```

Comment:
This is the dedicated network between the executor and the runtime?

Reply:
yes 👍